Ginger/GingerCoreNET/Logger/WebReportGenerator.cs

```diff
@@ -129,61 +130,48 @@ public void DeleteFoldersData(string clientAppFolderPath)
-            if (totalRunners != 0)
-                liteDbRunSet.ExecutionRate = string.Format("{0:F1}", (totalExecuted * 100 / totalRunners).ToString());
-            if (totalRunners != 0)
-                liteDbRunSet.PassRate = string.Format("{0:F1}", (totalPassed * 100 / totalRunners).ToString());
-            liteDbRunSet.Elapsed = liteDbRunSet.RunnersColl.Sum(a => a.Elapsed);
 
+            string imageFolderPath = Path.Combine(clientAppPath, "assets", "screenshots");
+            List<string> runSetEnv = new List<string>();
+
+            if (liteDbRunSet.ChildExecutableItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] != 0)
+                liteDbRunSet.ExecutionRate = string.Format("{0:F1}", (liteDbRunSet.ChildExecutedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] * 100 / liteDbRunSet.ChildExecutableItemsCount[_HTMLReportConfig.ExecutionCalculationLevel]).ToString());
+            if (liteDbRunSet.ChildExecutedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] != 0)
+                liteDbRunSet.PassRate = string.Format("{0:F1}", (liteDbRunSet.ChildPassedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] * 100 / liteDbRunSet.ChildExecutedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel]).ToString());
             foreach (LiteDbRunner liteDbRunner in liteDbRunSet.RunnersColl)
             {
                 if (!runSetEnv.Contains(liteDbRunner.Environment))
                 {
                     runSetEnv.Add(liteDbRunner.Environment);
                 }
-                int totalBFs = liteDbRunner.BusinessFlowsColl.Count;
-                int totalPassedBFs = liteDbRunner.BusinessFlowsColl.Where(bf => bf.RunStatus == eRunStatus.Passed.ToString()).Count();
-                int totalExecutedBFs = totalBFs - liteDbRunner.BusinessFlowsColl.Where(bf => bf.RunStatus == eRunStatus.Pending.ToString() || bf.RunStatus == eRunStatus.Skipped.ToString() || bf.RunStatus == eRunStatus.Blocked.ToString()).Count();
-                if (totalBFs != 0)
-                    liteDbRunner.ExecutionRate = string.Format("{0:F1}", (totalExecutedBFs * 100 / totalBFs).ToString());
-                if (totalExecutedBFs != 0)
-                    liteDbRunner.PassRate = string.Format("{0:F1}", (totalPassedBFs * 100 / totalExecutedBFs).ToString());
+                if (liteDbRunner.ChildExecutableItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] != 0)
+                    liteDbRunner.ExecutionRate = string.Format("{0:F1}", (liteDbRunner.ChildExecutedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] * 100 / liteDbRunner.ChildExecutableItemsCount[_HTMLReportConfig.ExecutionCalculationLevel]).ToString());
+                if (liteDbRunner.ChildExecutedItemsCount[_HTMLReportConfig.ExecutionCalculationLevel] != 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)